dataPipelines/gc_thumbnails/utils.py

```diff
@@ -0,0 +1,31 @@
+from pathlib import Path
+import fitz
+
+class ThumbnailsCreator:
+    def __init__(
+        self,
+        file_name,
```

Comment:
absent comments, would be good idea to type hints in the init method signature here, so `__init__(self, file_name: str, output_directory: str)` , or even more explicit: given `import typing as t`,  `def __init__(self, file_name: t.Union[str, Path], output_directory: t.Union[str, Path])` for full init function like:

```python
def __init__(self, file_name: t.Union[str, Path], output_directory: t.Union[str, Path]):
  self.file_name = Path(file_name).absolute()  # always a good idea to make your paths absolute so you're never confused by error messages referring to a relative path
  self.output_directory = Path(output_directory).absolute()   
```